package.json

```diff
@@ -25,7 +25,7 @@
     "packages/*",
     "packages/*/*",
     "examples/*",
-    "test-apps/*",
+    "test-apps/!(playwright)/**",
```

Comment:
So you're excluding the e2e app? will it use the monorepo version of the admin panel still?

Reply:
It does not. What I'd try as a next step once this PR is done is to link the dependencies to the folders in the monorepo as part of the `test:e2e:setup` step, after the installation (but outside of generators).

This step would at least unblock running the e2e, so that we can continue exploring parallelization e.g., but it doesn't solve the whole puzzle yet.